features/storage/nfs.feature

```diff
@@ -12,6 +12,7 @@ Feature: NFS Persistent Volume
     And admin creates a PV from "https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/storage/nfs/auto/pv-retain.json" where:
       | ["metadata"]["name"]      | nfs-<%= project.name %>          |
       | ["spec"]["nfs"]["server"] | <%= service("nfs-service").ip %> |
+    Given I ensure "nfsc" pvc is deleted after scenario
```

Comment:
IIUC, this teardown step will be run after the PV delete teardown step, then this case will be blocked, @liangxia Could you paste the test result here?

Reply:
Ah, sorry, it's added after pv is created, then lgtm.